src/DropdownButton/DropdownButton.less

```diff
@@ -58,35 +58,25 @@
  */
 
 .Button--primary.DropdownButton--toggle {
-  .DropdownButton--Caret--container {
-    .border--left--s(shade(@primary_blue, @shadeStepAmount * 2));
-  }
+  .border--left--s(shade(@primary_blue, @shadeStepAmount * 2));
```

Comment:
Turns out we need some of them. Or it doesn't show the line break for the primary or destructive.